liquibase-standard/src/main/java/liquibase/configuration/ConfiguredValue.java

```diff
@@ -33,8 +36,22 @@ public DataType getValue() {
         if (providedValue == null) {
             return null;
         }
+        return convertDataType(providedValue);
+    }
 
-        return valueConverter.convert(providedValue.getValue());
+    //
+    // Call the convert method with the argument key in the current scope
+    // so that it can be used in an error message
+    //
+    private DataType convertDataType(ProvidedValue providedValue) {
+        AtomicReference<DataType> reference = new AtomicReference<>();
+        try {
+            Scope.child("key", providedValue.getActualKey(), () ->
+                    reference.set(valueConverter.convert(providedValue.getValue())));
+            return reference.get();
+        } catch (Exception e) {
+            throw new IllegalArgumentException(e);
+        }
```

Comment:
Can you move this duplicated method into a static method in a utils class somewhere?

Reply:
I think I can refactor it to be a little more generic.  Then I can make it one method, instead of 3.